src/main/java/com/puppycrawl/tools/checkstyle/utils/CommonUtil.java

```diff
@@ -75,6 +75,9 @@
     /** Symbols with which multiple comment ends. */
     private static final String BLOCK_MULTIPLE_COMMENT_END = "*/";
 
+    /** Pseudo URL prefix for loading from the class path: "classpath:". */
+    private static final String CLASSPATH_URL_PREFIX = "classpath:";
```

Comment:
we need to add it to doc https://checkstyle.org/property_types.html#URI
website sources are in our repo, you can find them by grep.

Reply:
> we need to add it to doc https://checkstyle.org/property_types.html#URI
> website sources are in our repo, you can find them by grep.

I'll try:)
please help me fix the doc(ie: grammar)